en/airframes/adding_a_new_frame.md

```diff
@@ -1,25 +1,26 @@
-# Adding a new Airframe Configuration
+# Adding a New Airframe Configuration
 
 PX4 uses canned configurations as starting point for airframes. Adding a configuration is straightforward: Create a new file which is prepended with a free autostart ID in the [init.d folder](https://github.com/PX4/Firmware/tree/master/ROMFS/px4fmu_common/init.d) and [build and upload](../setup/building_px4.md) the software.
 
 Developers not wanting to create their own configuration can instead customize existing configurations using text files on the microSD card, as detailed on the [custom system startup](../advanced/system_startup.md) page.
 
-## Airframe configurations
+## Airframe Configurations
 
 An airframe configuration consists of three main blocks:
 
   * The apps it should start, e.g. multicopter or fixed wing controllers
-  * The physical configuration of the system (e.g. a plane, wing or multicopter). This is called mixer.
```

Comment:
@bkueng Just for my info, this is the mixer file - which you specify in the config file using `set MIXER wingwing` right?

Reply:
Right